addons/sale/static/src/js/product/product.xml

```diff
@@ -74,14 +76,14 @@
                     class="btn btn-secondary"
                     t-att-class="{'disabled': !this.env.isPossibleCombination(this.props)}"
                     t-on-click="() => this.env.addProduct(this.props.product_tmpl_id)">
-                    <i class="fa fa-plus"/> Add
+                    <i class="fa fa-plus d-none d-md-inline" role="img"/> Add
                 </button>
             </t>
         </td>
     </t>
 
     <t t-name="sale.product_price">
-        <h5 class="text-nowrap" t-out="getFormattedPrice()"/>
+        <span name="sale_product_configurator_formatted_price" class="h5 text-nowrap" t-out="getFormattedPrice()"/>
```

Comment:
Nit: in our team, we try to enforce a max line length of 100 characters. Would you mind splitting this into multiple lines? (you should be able to find some examples in this same file)